### DISCUSSION TITLE
Fdroid inclusion

### DISCUSSION THREAD

**Discussion by BoBeR182** [WrichikBasu/ShakeAlarmClock#62] [Feature requests]
Please deploy this app on fdroid.

**Comment by WrichikBasu** [comment]
I have received similar requests in the past. Unfortunately, I wasn't getting much time to work on the app. Now that a release is almost ready (a beta version has been released just now, see #63), I will go through the fdroid docs and try to incorporate the app.

Thank you for your interest and support!

**Comment by WrichikBasu** [comment]
Given that the app is now available on IzzyOnDroid (see readme for the link), I will probably not be hosting it on Fdroid separately.
